### ISSUE TITLE
extensions affect the returned avatar

### ISSUE THREAD

**Issue by rylnd** [adorableio/avatars-api-middleware#99]
We previously had code in place that made the requests `/avatars/foo.png` and `/avatars/foo`  "the same;" it was accomplished by simply stripping the extension out of the identifier. I removed this code because I thought it was silly, but I realize now that it's really a stopgap for #19.

Avatar-wise, the extension should _not_ affect the result. Long-term, it _should_ affect the content-type of the response.

The main reason to fix this sooner rather than later is that the .png toggle on the avatars site now behaves "weirdly," in that it actually changes the avatar being displayed.

**Comment by qaisjp** [comment]
Yeah the URL on the website gives a `.png` but the image displayed is without the `.png`
